Lib/test/test_urllibnet.py

```diff
@@ -24,8 +24,8 @@ def tearDown(self):
         socket.setdefaulttimeout(None)
 
     def testURLread(self):
-        with support.transient_internet("www.example.com"):
-            f = urllib.request.urlopen("http://www.example.com/")
+        with support.transient_internet(support.TEST_HOSTNAME):
```

Comment:
Would it be possible to extract the hostname from the URL rather than introducing another constant? It would avoid the risk of avoid 2 constants which would be inconsistent (if 1 is updated but not the other).

urllib.parse has tools for that.

Reply:
yep, we can do it. I am going to fix it.